### PR TITLE
Fix variable reference error in 'ineffective-segments' lint

### PR DESCRIPTION
**Pull request by mondeja**
Sorry for this error, my bad. This was introduced in #4427: `segments` and `absSegments` variables has the same reference and this is causing that reported errors are wrong.

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
LGTM! Will wait on another @simple-icons/maintainers before merging though.

**Review by service-paradis — APPROVED** [review]
Good catch! Missed it while reviewing the other PR, thanks!

**Pull request merged**
